skimage/novice/_novice.py

```diff
@@ -282,6 +282,7 @@ def from_size(size, color='black'):
             color = color_dict[color]
         rgb_size = tuple(size) + (len(color),)
         array = np.ones(rgb_size, dtype=np.uint8) * color
+        array = array.astype(np.uint8)
```

Comment:
I think it makes more sense to update the previous line?
```python
array = np.ones(rgb_size, dtype=np.uint8) * np.array(color, dtype=np.uint8)
```
Prevents the creation of an 8x bigger array. Not that performance will typically be an issue here but it bugs me. =)